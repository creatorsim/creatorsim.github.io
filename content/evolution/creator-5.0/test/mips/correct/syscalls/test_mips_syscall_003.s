#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:
	# print int
	li $v0, 3
	li.d $FP12, 6673
	syscall

  	# print negative double
    li.d $FP12, -86.974
    syscall
    
    li.d $FP12, 1.1
    syscall
    