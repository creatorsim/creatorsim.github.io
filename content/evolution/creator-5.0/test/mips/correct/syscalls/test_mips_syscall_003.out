
6673
-86.974
1.1
cr[PC]:0x58; ir[1,at]:0x3ff19999; ir[2,v0]:0x3; sfpr[f12]:0x3FF19999; sfpr[f13]:0x9999999A; dfpr[FP12]:0x3FF199999999999A; keyboard[0x0]:''; display[0x0]:'6673-86.9741.1'; 

